tests/actiontest.cpp

```diff
@@ -169,9 +152,9 @@ private slots:
         QCOMPARE(a1->isChecked(), false);
         QCOMPARE(changeSpy.size(), 2);
     }
-
 };
 
 QTEST_MAIN(ActionTest)
 
+#include "moc_baseprobetest.cpp"
```

Comment:
Is that distinction actually necessary? or is that just because one test was written somewhat differently for historic reasons?